src/common/stream.h

```diff
@@ -93,6 +93,28 @@ typedef __int128_t hge;
 #define STREAM_FWF_RECORD_SEP '\n'
 #define STREAM_FWF_FILLER ' '
 
+#ifdef _MSC_VER
```

Comment:
What's the point of moving this to the header?

Reply:
Maybe I can move it to the implementation file. Yes!

Reply:
I guess from the API point of view it's more organized